frontend/src/components/LoginPage.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react";
+import TUMLogo from "../assets/tum-logo.svg"; // 确保路径正确
+
+const LoginPage = () => {
+    const [email, setEmail] = useState("");
+    const [error, setError] = useState("");
+
+    const handleSignIn = () => {
+        const emailRegex = /^[^\\s@]+@[^\\s@]+\\.[^\\s@]+$/;
+        if (!emailRegex.test(email)) {
+            setError("Invalid Account, please try again!");
+        } else {
+            setError("");
+            console.log("Sign-in with email:", email);
+        }
+    };
+
+    return (
+        <div
+            className="relative flex justify-center items-center min-h-screen"
+            style={{
+                backgroundColor: "#f5f5f5", // 设置页面背景色
+            }}
+        >
+            {/* 左上角 TUM Logo */}
+            <div
+                style={{
+                    position: "absolute",
+                    top: "10px",
+                    left: "30px",
+                    zIndex: 100, // 确保 Logo 在最上层
```

Comment:
NIT: It's better to use relative measurement such as 'rem' or 'em' instead of hard code like 'px'. It better for RWD though we didn't do that. Tailwind has the classname of then, e.g.,  left-6 means left-[30px]